xtra-libp2p-ping/src/lib.rs

```diff
@@ -95,10 +104,39 @@ mod tests {
             [(PROTOCOL_NAME, pong_address.clone_channel())],
         );
 
+        let (supervisor, _connection) = supervisor::Actor::with_policy(
+            {
+                let ping_address = ping_address.clone();
+                let endpoint_address = endpoint_address.clone();
+                move || {
+                    endpoint_monitor::Actor::new(
+                        endpoint_address.clone(),
+                        endpoint_monitor::Subscribers::new(
```

Comment:
🙃 nit: For readability outside of an IDE it would actually be better to have just the `struct` I think because then you see which field is set to what.

i.e.:

```
                          endpoint_monitor::Subscribers {
                            connection_established: vec![xtra::message_channel::MessageChannel::clone_channel(
                                &ping_address,
                            )],
                            connection_dropped: vec![xtra::message_channel::MessageChannel::clone_channel(
                                &ping_address,
                            )],
                            listen_address_added: vec![],
                            listen_address_removed: vec![],
                        },
```